src/Nethermind/Nethermind.Shutter/Config/IShutterConfig.cs

```diff
@@ -11,14 +12,9 @@ namespace Nethermind.Shutter.Config;
 
 public interface IShutterConfig : IConfig
 {
-    // todo: replace with bootnodes when peer discovery added
     private const string DefaultP2PAddresses =
-@"/ip4/139.59.130.109/tcp/23003/p2p/12D3KooWRZoofMsnpsjkgvfPQUyGXZQnn7EVnb4tw4ghNfwMnnsj,
-/ip4/167.71.169.248/tcp/23003/p2p/12D3KooWGH3VxoSQXZ6wUuCmsv5caGQnhwfGejbkXH6uS2r7sehA,
-/ip4/139.59.130.109/tcp/23003/p2p/12D3KooWNxTiw7CvD1fuyye5P8qPhKTTrRBW6wwZwMdqdTxjYF2H,
-/ip4/178.128.192.239/tcp/23003/p2p/12D3KooWCdpkipTiuzVMfkV7yLLgqbFeAL8WmEP78hCoBGBYLugN,
-/ip4/45.55.192.248/tcp/23003/p2p/12D3KooWMPuubKqksfMxvLwEBDScaopTdvPLr5J5SMmBEo2zkcMz,
-/ip4/178.128.126.237/tcp/23003/p2p/12D3KooWAg1pGUDAfFWSZftpN3JjBfLUCGLQcZApJHv2VntdMS9U";
+@"/ip4/167.99.177.227/tcp/23003/p2p/12D3KooWD35AESYCttDEi3J5WnQdTFuM5JNtmuXEb1x4eQ28gb1s,
+/ip4/159.89.15.119/tcp/23003/p2p/12D3KooWRzAhgPA16DiBQhiuYoasYzJaQSAbtc5i5FvgTi9ZDQtS";
```

Comment:
Should this be in code? Rather than in config files?

Reply:
These do not change depending on the network?

Reply:
Yes, it defaults to gnosis but is overridden for chiado: https://github.com/NethermindEth/nethermind/blob/master/src/Nethermind/Nethermind.Runner/configs/chiado.cfg#L34

Reply:
Why not handle it the same way as other config values and have it defined in the config file?

Reply:
I don't mind either way, maybe it's a bit easier to make it default. I changed it because of this review comment: https://github.com/NethermindEth/nethermind/pull/7414#discussion_r1758950890

Reply:
Hm.. my comment was do we want it in default `gnosis.cfg` and not as defaults for all config :)

Reply:
Ah I see what you mean, I think it makes sense to have in `gnosis.cfg` then, since it won't do anything unless the `Shutter.Enabled` flag is true